local-volume/provisioner/deployment/kubernetes/gce/provisioner_generated_gce_pre19.yaml

```diff
@@ -0,0 +1,56 @@
+---
+# Source: provisioner/templates/provisioner.yaml
+ 
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: local-provisioner-config 
+  namespace: default 
+data:
+  storageClassMap: |     
+    local-scsi:
+       hostDir: /mnt/disks
```

Comment:
Could this yaml ever be run with post 1.9 and with local SSD volume configs? If so, these would pick up the entire directory including the new "by-uuid" directory, so we would have *double* the mounts.

Not sure if the provisioner only scans top level dir. Just want to make sure.

Reply:
Ooh interesting, I will try this out.  Looking at the code, it only looks at the top level directory, and if the file is not a mount point, it will be ignored.  So the worst case is that the provisioner will just be periodically logging that this "by-uuid" directory is not a mountpoint.

Reply:
Yeah just tried it out, so we will not create double the PVs, but there are 2 minor issues:
1. Provisioner will be logging "Path "/mnt/disks/by-uuid" is not an actual mountpoint" every 10 seconds
2. All the scsi and nvme PVs will be created under the "local-scsi" storageclass since they are all in the same directory.

If we eventually add these as addons, hopefully that can help eliminate these potential user errors.

Reply:
to me these seem like acceptable "failure modes" due to user error. Agree that making these addons would solve this issue.